src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,13 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    transition-duration: 300ms;
```

Comment:
1) add transition properties to those rules that you'd like to change (transform, color)
2) create a variable for transition-duration value to not repeat yourself